### PR TITLE
Be consistent with page names, titles and menu

### PR DESCRIPTION
**Pull request by tpmccallum**
Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [x] The `title, `template`, and `date` are all set
- [x] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [x] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result) 
- [x] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)

✅ content/spin/ai-sentiment-analysis-api-tutorial.md  
✅ content/spin/key-value-store-tutorial.md   
✅ content/spin/registry-tutorial.md
✅ content/spin/url-shortener-tutorial.md

- [x] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)
- [x] Headings are using Title Case
- [x] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [x] Have tested with `npm run test` and resolved all errors
- [x] Relates to an existing (potentially outdated) blog article? If so please add URL in blog to point to this content.

### CODE REVIEW

File: templates/spin_sidebar.hbs [hunk 1]
```diff
@@ -106,16 +106,14 @@
...
-                    <li><a {{#if (active_content request.spin-path-info "/spin/url-shortener" )}} class="active" {{/if}}
-                            href="{{site.info.base_url}}/spin/url-shortener">Building a URL Shortener With Spin</a></li>
-                    <li><a {{#if (active_content request.spin-path-info "/spin/spin-oci" )}} class="active" {{/if}}
-                            href="{{site.info.base_url}}/spin/spin-oci">Spin Apps in Registries</a></li>
-                    <li><a {{#if (active_content request.spin-path-info "/spin/kv-store-tutorial" )}} class="active"
-                            {{/if}} href="{{site.info.base_url}}/spin/kv-store-tutorial">Persistent Data Locally With
-                            Key Value
-                            Store</a></li>
+                    <li><a {{#if (active_content request.spin-path-info "/spin/ai-sentiment-analysis-api-tutorial" )}} class="active"
+                            {{/if}} href="{{site.info.base_url}}/spin/ai-sentiment-analysis-api-tutorial">AI Sentiment Analysis API Tutorial</a></li>
```
Comment (itowlson) [inline templates/spin_sidebar.hbs:110]: I question whether we need to say "Tutorial" in the TOC when this is already in the Tutorials section?

**Review by itowlson — CHANGES_REQUESTED** [review]
Great to see these titles being tightened up and made more consistent.  Unfortunately there are a couple of places where the proposed new title doesn't quite work for me - sorry - hopefully my concerns make sense.  I'm happy to brainstorm with you on alternatives if that would be useful.

File: templates/spin_sidebar.hbs (hunk 1 above)
Comment (tpmccallum) [reply templates/spin_sidebar.hbs:110]: Yeah, good point. 
I will remove the word “Tutorial” from the visible/clickable menu text.

Comment (tpmccallum) [reply templates/spin_sidebar.hbs:110]: I also think the same goes for the fact that the user is on the “Spin” project; and each page has the `/spin` location in the URLs path (which is why I removed any `Spin` prefixes from the visible/clickable menu text).

Comment (itowlson) [reply templates/spin_sidebar.hbs:110]: Yes, good call.  I wonder if the registry one would then go to "Storing Applications in Registries" and the URL shortener one to something like "Building a URL Shortener" perhaps?  I think they need that bit of extra context but agree we can drop the Spin part.

File: content/spin/url-shortener-tutorial.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-title = "Building a URL shortener with Spin"
+title = "URL Shortener Tutorial"
```
[the remark this answers is not in the document]
Comment (tpmccallum) [reply content/spin/url-shortener-tutorial.md:1]: Understand.
These have been updated in the next push.

[the remark this answers is not in the document]
Comment (tpmccallum) [reply content/spin/url-shortener-tutorial.md:1]: We are in the `spin` project and under the `tutorial` menu heading, but I guess sprinkling some of that back into the menu’s title does work (just trying to keep the wordiness to a minimum though). I am sure you would agree :)

File: content/spin/registry-tutorial.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
-title = "Spin Apps in Registries"
+title = "Registry Tutorial"
```
[the remark this answers is not in the document]
Comment (tpmccallum) [reply content/spin/registry-tutorial.md:1]: Updated in the next push.

File: content/spin/ai-sentiment-analysis-api-tutorial.md [hunk 4]
```diff
@@ -1,9 +1,9 @@
-title = "Serverless AI Tutorial"
+title = "AI Sentiment Analysis API Tutorial"
```
[the remark this answers is not in the document]
Comment (tpmccallum) [reply content/spin/ai-sentiment-analysis-api-tutorial.md:1]: Understand!
Had a rethink and have added new changes in latest push.

**Comment by tpmccallum** [comment]
All makes sense; just a general tidy-up and I’m happy to be flexible on the title names.

File: content/spin/ai-sentiment-analysis-api-tutorial.md (hunk 4 above)
Comment (tpmccallum) [inline content/spin/ai-sentiment-analysis-api-tutorial.md:1]:
```suggestion
title = "Sentiment Analysis With Serverless AI"
```

File: content/spin/url-shortener-tutorial.md (hunk 2 above)
Comment (tpmccallum) [inline content/spin/url-shortener-tutorial.md:1]:
```suggestion
title = "Building a URL Shortener With Spin"
```

File: content/spin/registry-tutorial.md (hunk 3 above)
Comment (tpmccallum) [inline content/spin/registry-tutorial.md:1]:
```suggestion
title = "Spin Apps in Registries"
```

File: content/spin/key-value-store-tutorial.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
-title = "Persistent Data Locally With Key Value Store"
+title = "Key Value Store Tutorial"
```
Comment (tpmccallum) [inline content/spin/key-value-store-tutorial.md:1]:
```suggestion
title = "Spin Key-Value Store"
```

**Review by tpmccallum — COMMENTED** [review]
Update page title to match menu text.

**Review by itowlson — APPROVED** [review]

**Pull request merged**